tests/unit/h_vialib/secure/secrets_test.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from Cryptodome.Random import get_random_bytes
+from Cryptodome.Util.Padding import pad
+
+from h_vialib.secure import SecureSecrets
+
+
+class TestSecureSecrets:
+    def test_encrypt_dict_round_trip(self, secure_secrets):
```

Comment:
The whole process has a lot of conversions back and forwards so this tests is very useful.

The rest are there to jsut assert that we are in fact calling AES.